### ISSUE TITLE
Exposure compensation from Exif is not applied as Exposure correction automatically

### ISSUE THREAD

**Issue by hholtm** [Beep6581/RawTherapee#5833]
If exposure compensation is read from exif while loading a picture, a compensation of 0 i read as 0. But if the compensation value is less than 0, RT always shows -1.00, regardless to the actual value of exif-data. I think during loading a picture without .pp3 the exposure compensation should be the same as in exif-data. 
Can someone please check and confirm when indicated.
I'm using Appimage RT-dev 5.8-2276-g18d3f1c on Linux Mint 19.3.

**Comment by heckflosse** [comment]
works fine here
![grafik](https://user-images.githubusercontent.com/13733487/86532933-cae80d80-becd-11ea-8a5d-74025d196932.png)

**Comment by Thanatomanic** [comment]
Can you share a raw file (+pp3) that you think is affected?

**Comment by hholtm** [comment]
Info as shown by @heckflosse is ok here too, but the EV-value is not transferred to Exposure compensation inside Exposure. And I thought this value should be transferred by first loading of a picture (without existing .pp3). Or is my presumption simply wrong?

**Comment by hholtm** [comment]
![Info](https://user-images.githubusercontent.com/39562391/86536378-4e622880-bee7-11ea-900a-f97bcecdf8e5.png)

Info (EV: 0.3)
![EC](https://user-images.githubusercontent.com/39562391/86536384-57eb9080-bee7-11ea-91e6-c2d78cd2a688.png)

Exposure compensation (-1.00)

**Comment by heckflosse** [comment]
> Or is my presumption simply wrong?

Yes, your presumption is wrong. The EV-value is not used automatically, except you use `dynamic profiles` to do that
![grafik](https://user-images.githubusercontent.com/13733487/86536363-27a3f200-bee7-11ea-9341-ba85843813df.png)

**Comment by heckflosse** [comment]
Your copy/paste didn't work

**Comment by hholtm** [comment]
@heckflosse Thank you very much! I didn't know that.

**Comment by heckflosse** [comment]
@hholtm If you want to apply automatic correction for ev, just make a bunch of dynamic profiles. For example one which just sets the exposure compensation to + 0.3 like this
[apply_expcomp_p03.zip](https://github.com/Beep6581/RawTherapee/files/4875195/apply_expcomp_p03.zip)
and define some dynamic profiles like this one for files with -0.3 EV
![grafik](https://user-images.githubusercontent.com/13733487/86536613-44d9c000-bee9-11ea-8897-9dfc8871defe.png)

For reference:
http://rawpedia.rawtherapee.com/Dynamic_processing_profiles

**Comment by Thanatomanic** [comment]
What do you think about a button near the Exposure slider that can be used to set the Exposure slider to a value that compensates for the value in the Exif? Or maybe a toggle button?

**Comment by heckflosse** [comment]
> What do you think about a button near the Exposure slider that can be used to set the Exposure slider to a value that compensates for the value in the Exif? Or maybe a toggle button?

I would appreciate this :+1:

**Comment by dimonoid** [comment]
I think the string is not converted to float or something...
Same thing with focal length
